vmdb/app/models/metric/ci_mixin/capture/openstack_base.rb

```diff
@@ -37,134 +37,355 @@ def perf_release_openstack
   end
 
   def perf_capture_data_openstack_base(metric_capture_module, start_time, end_time, resource_filter, metadata_filter)
-    # some meters can get gathered by directly querying for the "resource_id",
-    #   but other meters can only be gathered by examining the
-    #   "metadata.instance_id"
     log_header = "MIQ(#{self.class.name}.perf_collect_data_openstack) [#{start_time} - #{end_time}]"
-    $log.debug "#{log_header} start_time: #{start_time}, end_time: #{end_time}"
+    $log.debug "#{log_header} id:[#{name}] start_time: #{start_time}, end_time: #{end_time}"
```

Comment:
I missed this in your last PR to consolidate the code into the base ci_mixin.  Can you update the `log_header` here.  Right now it will output `perf_collect_data_openstack`, but that hides the fact that `openstack_infra` calls this, too.

I'm not sure whether it's better to output just `perf_capture_data_openstack_base`, or whether it's better to figure out who the caller is and show something indicating the caller as well.

This comes into play later during `process_multi_counter_stats`, where it can warn that the data is corrupted possibly due to the fact that Ceilometer is overloaded.